x-pack/docs/en/security/securing-communications/configuring-tls-docker.asciidoc

```diff
@@ -206,9 +206,6 @@ WARNING: Windows users not running PowerShell will need to remove `\` and join l
 ----
 docker exec es01 /bin/bash -c "bin/elasticsearch-setup-passwords \
 auto --batch \
--Expack.security.http.ssl.certificate=certificates/es01/es01.crt \
--Expack.security.http.ssl.certificate_authorities=certificates/ca/ca.crt \
--Expack.security.http.ssl.key=certificates/es01/es01.key \
```

Comment:
Thanks! This works, however -- at least this PR -- includes the [UBI work](https://github.com/elastic/elasticsearch/pull/48710) and [earlier in this file](https://github.com/elastic/elasticsearch/pull/49612/files#diff-13bed7d96a74500ddb18bc537d198109L74) we rely on `yum install unzip`. In order to simply things, given that `unzip` is a fairly common requirement for handling certs, what if we include `unzip` in our `Dockerfile` i.e.bundle it in all Elasticsearch images and remove it from the the mentioned line above, so that examples and users don't have to pull it?

Reply:
That makes sense to me, and we already include `gzip`.

Reply:
Great. Maybe, esp. given that we have abstracted the package manager invocation now, we could test for the present of `unzip` / `zip` in the path in our test cases, if it's not too complex?

Reply:
Good call on the test - I was installing the CLI tools in the wrong part of the Dockefile!